OrbitGl/CaptureDeserializer.cpp

```diff
@@ -143,24 +101,80 @@ ErrorMessageOr<void> CaptureDeserializer::Load(std::istream& stream) {
     ERROR("%s", error_message);
     return ErrorMessage(error_message);
   }
-  CaptureData capture_data = GenerateCaptureData(capture_info);
+  time_graph->Clear();
```

Comment:
Thanks for moving this out of GenerateCaptureData

Reply:
No problem! :)